www/js/survey/launch.js

```diff
@@ -64,8 +64,14 @@ angular.module('emission.survey.launch', ['emission.services',
                                                          endFmtTimeElementId,
                                                          startTs,
                                                          endTs) {
+      var options = {
+        location: 'no',
+        clearcache: 'no',
+        toolbar: 'yes'
+      };
+
       // THIS LINE FOR inAppBrowser
-      $cordovaInAppBrowser.open(url, '_blank', surveylaunch.options)
+      $cordovaInAppBrowser.open(url, '_blank', options)
```

Comment:
why revert this?

Reply:
I think this and the thing above might have just gotten deleted, I can put it back.